xio-core/src/main/java/com/xjeffrose/xio/config/ConfigReloader.java

```diff
@@ -72,24 +87,60 @@ private Config parse(Reader reader) {
     return load(new File(file));
   }
 
-  private boolean performUpdate(Meta<T> oldValue, Meta<T> newValue) {
-    return !MessageDigest.isEqual(oldValue.digest, newValue.digest);
+  private boolean performUpdate(ConfigFileMetadata current, ConfigFileMetadata previous) {
```

Comment:
these new parameter names are not as clear (and swapped with previous impl) - `oldValue` / `newValue`

Reply:
ok i can change the name

Reply:
name changed.